src/dom/js/selector-css2.js

```diff
@@ -74,7 +74,14 @@ var PARENT_NODE = 'parentNode',
                 className = token.className;
                 tagName = token.tagName || '*';
 
-                if (root.getElementsByTagName) { // non-IE lacks DOM api on doc frags
+                var getElementsByTagName = true;
+
+                try {
+                    getElementsByTagName = root.getElementsByTagName;
```

Comment:
ie11 always wants to run "root.getElementsByTagName" as a function. I think it does it because root is a [IXMLDOMDocument2] object.